R/extract_features.R

```diff
@@ -421,6 +421,9 @@ input_fn_builder_EF <- function(features,
 #' potentially be used as features in downstream tasks.)
 #'
 #' @param examples List of \code{InputExample_EF}s to convert.
+#' @param ckpt_dir Character; path to checkpoint directory. If specified,
```

Comment:
Remove ckpt_dir documentation from this file. Find/replace cp_dir in utils.R to ckpt_dir. Add `@inheritParams find_vocab` to this one.

Likewise, inheritParams FullTokenizer (which should inheritParams load_vocab).

bert_config_from_json_file should rename its arg to bert_config_file, and then extract_features should inheritParams from bert_config_from_json_file.

init_checkpoint should be documented in one place, too, and then the other things should inherit. I'm not sure if `.model_fn_builder_EF` should be the "home", or one of the other functions.

Basically... I really really think we should focus on documenting each repeated-use argument in ONE place (unless it has subtly different meanings in different places), to avoid headaches when we find a typo in one of them, etc.

Reply:
I completely agree with this idea. In fact, I had originally used `inheritParams` in `tokenize_text` instead of repeating the documentation. However,  I then realized that the documentation was slightly different (in mentioning just the vocab vs. all three bits of the checkpoint). So I need to think a bit about how to document so that it's "portable" without losing helpful detail (I've almost got it).

It also sounds like there's a lot more documentation clean-up we should do, which should probably be part of its own PR. I'll clean up the args for the functions added in this PR, though.